Controller/Admin/TransactionTabAccountOwner.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Wirecard\Oxid\Controller\Admin;
+
+use Wirecard\Oxid\Model\Transaction;
+use Wirecard\Oxid\Controller\Admin\TransactionTab;
+
+use OxidEsales\Eshop\Application\Model\Order;
+use OxidEsales\Eshop\Application\Model\Payment;
+
+/**
+ * Controls the view for the account owner transaction tab.
+ */
+class TransactionTabAccountOwner extends TransactionTab
+{
+    /**
+     * @inheritdoc
+     *
+     * @param Transaction $transaction
```

Comment:
Please **remove the params & return value** as they seem to be exact duplicates of the parent documentation.

Reply:
Can't do, our PHPCS linter will fail then.